mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/VirksomhetService.kt

```diff
@@ -24,9 +28,24 @@ class BrregService(private val brregClient: BrregClient) {
     }
 
     suspend fun hentEnhet(orgnr: String): VirksomhetDto {
```

Comment:
Kanskje rename denne til å reflektere at det i praksis kan skje to ting når man kaller denne metoden. Man kan både hente, men det skjer også en sideeffekt ved oppdatering til db her. Det er kanskje ryddigere å splitte det ut i to metoder så det blir mer forståelig for den som leser koden?